tests/src/TravisLogEventListener.php

```diff
@@ -17,39 +17,39 @@ class TravisLogEventListener implements TestListener {
    */
   protected $errors;
 
-  public function addWarning(Test $test, Warning $e, $time) {
+  public function addWarning(Test $test, Warning $e, float $time): void {
     $this->errors = TRUE;
     file_put_contents(TRAVIS_BUILD_DIR . '/solr.error.log', printf("Warning while running test '%s'.\n", $test->getName()), FILE_APPEND | LOCK_EX);
   }
 
-  public function addError(Test $test, \Exception $e, $time) {
+  public function addError(Test $test, \Throwable $e, float $time): void {
```

Comment:
Does that work for Drupal 8.8 and 9.0 in parallel? In other words could we use PHPUnit 7 for Drupal 8.8?

Reply:
The tests should prove that, but surprisingly, I think so. I didn't see any errors from phpstorm when having this applied on a D8 site. But if not, since this is only for travis, you could keep the changes or even the whole file in a travis-only patch and apply it based on the core version. Running the tests against all versions again should verify that.

Reply:
It works :-)